server/controllers/airTableController.js

```diff
@@ -0,0 +1,46 @@
+import regeneratorRuntime from "regenerator-runtime";
```

Comment:
Out of curiosity, do you need to import this? I thought `babel` was able to do it itself through a preset.